contracts/api3-server-v1/AuctioneerRegistry.sol

```diff
@@ -17,18 +17,17 @@ contract AuctioneerRegistry is Ownable, SelfMulticall {
 
     // This fee shouldn't guarantee indefinite support. We can specify that we
     // will deactivate proxies that haven't generated any OEV for a length of
-    // time.
-    // Similarly, we should be able to deactivate proxies whose metadata is not
-    // formatted correctly, or ones that are on chains that we're not serving
-    // yet, etc. Payment should not be reason to allow these to take up
+    // time. Similarly, we should be able to deactivate proxies whose data is
+    // not formatted correctly, or ones that are on chains that we're not
+    // serving yet, etc. Payment should not be reason to allow these to take up
     // auctioneer capacity.
     uint256 public activationFeeInUsdWith18Decimals;
 
     uint256 public maximumOevProxyCount;
 
-    mapping(bytes32 => bytes) public hashToOevProxyMetadata;
+    mapping(bytes32 => bytes) public hashToOevProxyData;
```

Comment:
I think I will call this "OEV proxy details" as in "data feed details" from AirseekerRegistry because they are quite similar